app/src/main/java/com/swent/mapin/ui/map/BottomSheetContent.kt

```diff
@@ -313,42 +380,66 @@ fun BottomSheetContent(
 private fun SearchResultsSection(
     results: List<Event>,
     query: String,
+    recentItems: List<RecentItem> = emptyList(),
+    onRecentSearchClick: (String) -> Unit = {},
+    onRecentEventClick: (String) -> Unit = {},
+    onShowAllRecents: () -> Unit = {},
+    topCategories: List<String> = emptyList(),
+    onCategoryClick: (String) -> Unit = {},
+    filterViewModel: FiltersSectionViewModel,
+    locationViewModel: LocationViewModel,
+    userProfile: com.swent.mapin.model.UserProfile? = null,
```

Comment:
Maybe import the package on top of the file to make it A LOT cleaner
```suggestion
    import com.swent.mapin.model.UserProfile

    userProfile: UserProfile? = null,
```